Assets/Scripts/Common/Extensions/UIBehaviourExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static class UIBehaviourExtensions
     public static void SetPositionOnScreen(this UIBehaviour uiElement, Vector3 position)
     {
       uiElement.transform.position = position.z < 0 
-        ? Vector3.zero 
+        ? Vector3.one * -200 //far outside screen
```

Comment:
I would keep the z coord at 0 to avoid bugs later on.

Reply:
but labels at 0 are still visible at bottom left corner